app/controllers/workflow_repository_controller.rb

```diff
@@ -30,12 +95,64 @@ def show
     super
   end
 
+  def show_output
+    drop_breadcrumb(:name => _("Refresh output"), :url => show_output_link)
+    @showtype = 'output'
+  end
+
+  def display_workflows
+    nested_list(ManageIQ::Providers::Workflows::AutomationManager::Workflow, :breadcrumb_title => _('Workflows'))
+  end
+
+  def repository_refresh
+    assert_privileges("embedded_configuration_script_source_refresh")
+    checked = find_checked_items
+    checked[0] = params[:id] if checked.blank? && params[:id]
+
+    WorkflowRepositoryController.model.where(:id => checked).each do |repo|
```

Comment:
```suggestion
   model.where(:id => checked).each do |repo|
```

Reply:
Weird model is defined here https://github.com/ManageIQ/manageiq-ui-classic/pull/8842/files#diff-739ffa064264fde3da20fb632fc13abc52e6b12009d5d015e195fe65014a4ce5R23-R25

Maybe `self.class.model.where(:id => checked)` ?

Reply:
Still hitting the same error

Reply:
Well that doesn't make any sense... `self.class == WorkflowRepositoryController` here (at least it should)

Reply:
Oh yeah it is working, just tried it again and it seems good. Not sure why it didn't work the first time.